CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Routing
 
 * Added `Router.finishRouting()` method to finish routing session without dismissing related UI and logic components. ([#3880](https://github.com/mapbox/mapbox-navigation-ios/pull/3880))
+* Added reporting `AlternativeRoutes` during navigation sessions. Subscribe to `AlternativeRoutesCenter` to receive updates on actual alternative routes list. To configure alternative routes detection, setup `NavigationSettings.AlternativeRouteDetectionOptions` before starting navigation. ([#3833](https://github.com/mapbox/mapbox-navigation-ios/pull/3833))
```

Comment:
It’s actually called `NavigationSettings.alternativeRoutesOptions`, but `alternativeRouteDetectionOptions` sounds a lot better and would be consistent with the type name.